### PR TITLE
Fixed: Confusing Notification Message for Non-HTTP Probe

### LINKED ISSUE #1062: Wrong default alerts for non-HTTP probe
**Describe the bug**

When probing a non-http probe, like postgres, without specifying the alerts, e.g. using the following config:

```yaml
probes:
  - id: 'postgres-01'
    name: database health
    description: ensure db health
    interval: 10 # in seconds
    incidentThreshold: 3
    postgres:
      - host: 127.0.0.1
        port: 5432
        user: user
        password: password
        database: mydb
```

Monika "sanitizes" the probe by adding some default values including the `alerts`. However, the default value for the `alerts` don't make sense for non-HTTP probe, namely Monika will add HTTP-related alerts like the following:

```javascript
{
  id: 'postgres-01',
  name: 'database health',
  description: 'ensure db health',
  interval: 10,
  incidentThreshold: 3,
  postgres: [
    {
      host: '127.0.0.1',
      port: 5432,
      user: 'user',
      password: 'password',
      database: 'mydb'
    }
  ],
  requests: [],
  alerts: [
    {
      assertion: 'response.status < 200 or response.status > 299',
      message: 'HTTP Status is {{ response.status }}, expecting 200'
    },
    {
      assertion: 'response.time > 2000',
      message: 'Response time is {{ response.time }}ms, expecting less than 2000ms'
    }
  ],
  recoveryThreshold: 5
}
```

This results in weird incident notification message, e.g., in Slack:

Screenshot 2023-06-14 at 22 56 32

**To Reproduce**
1. Use the config I mentioned above.
2. Run monika with that config.
3. See the unrelated notifications.

**Expected behavior**
A clearer and related incident/recovery messages.

**Desktop (please complete the following information):**
 - OS: macOS Ventura 13.4
 - Version commit 3bb855b6bfe4335106968b046bfe7a609a9bec88

**Additional context**
I only checked the postgres but pretty sure this bug applies to all non-http probe.

### PR DESCRIPTION
**Pull request by haricnugraha**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
Fix confusing notification message for non-HTTP probe. It will resolves #1062. 

## How did you implement / how did you fix it  
1. Add docker-compose file to make it easier to test non-HTTP probe
2. Change the default alerts for non http probe
3. Change URL to Probe ID in the notification message
4. Add test

## How to test  
1. Run Postgres on Docker. Run `cd dev && docker compose up`
2. Run Monika with the following config
3. Stop the Postgres from the step 1 `cmd/ctrl + c`
4. Run the Postgres after received the incident notification

```yaml
probes:
  - id: postgres
    interval: 3
    incidentThreshold: 1
    recoveryThreshold: 1
    postgres:
      - uri: postgres://postgres_user:postgres_password@localhost:5432/postgres_db
```

## Video
https://github.com/hyperjumptech/monika/assets/15191978/38a5f0df-faf1-4395-96f1-26442bb843c6

### CODE REVIEW

File: .dev/docker-compose.yaml
Comment (nicnocquee) [inline .dev/docker-compose.yaml:1]: better to make this `dev` dir visible

**Review by nicnocquee — APPROVED** [review]
Please add how to use the docker-compose in the readme.

File: .dev/docker-compose.yaml
Comment (haricnugraha) [reply .dev/docker-compose.yaml:1]: Thanks for the review. Moved the directory [here (3d49b8c)](https://github.com/hyperjumptech/monika/pull/1072/commits/3d49b8cbcfbd1ddba93b89f4b37d0e732e0f164f) and added the documentation [here (d631050)](https://github.com/hyperjumptech/monika/pull/1072/commits/d631050db06ac890029f932672f9a3a326090cb1).

**Review by sapiderman — APPROVED** [review]
Good PR! 🥇 🏆 

before pr:
image

after pr:
image

**Pull request merged**